src/components/organisms/pupil/OakPupilJourneyUnitsFilter/OakPupilJourneyUnitsFilter.tsx

```diff
@@ -1,53 +1,69 @@
 import React from "react";
-import styled from "styled-components";
 
-import {
-  OakButtonAsRadioGroup,
-  OakOutlineAccordion,
-  OakSecondaryButtonAsRadio,
-} from "../../../molecules";
-import { OakBox, OakHeading } from "../../../atoms";
+import { OakButtonAsRadioGroup } from "@/components/molecules/OakButtonAsRadioGroup";
+import { OakSecondaryButtonAsRadio } from "@/components/molecules/OakSecondaryButtonAsRadio";
+import { OakOutlineAccordion } from "@/components/molecules/OakOutlineAccordion";
+import { OakHeading } from "@/components/atoms/OakHeading";
+import { OakFlex } from "@/components/atoms";
 
-export type menuItem = {
-  text: string;
-  id: number;
+type MenuItem = {
+  displayText: string;
+  value: string;
 };
 
 export type OakPupilJourneyUnitsFilterProps = {
-  menuItems: menuItem[];
-  selected: number;
-  onSelected: (arg0: menuItem) => void;
+  menuItems: MenuItem[];
+  selected: string;
+  onSelected: (arg0: MenuItem) => void;
 };
 
-const UnstyledComponent = (props: OakPupilJourneyUnitsFilterProps) => {
-  const { menuItems, selected, onSelected, ...rest } = props;
+/**
```

Comment:
I'm not sure why we were exporting this as styled component. It seems unlikely that it would ever need to be "inherited" . We should only really use that pattern when it is actually needed.